packages/fluent-bit/extra/dcos-fluent-bit.service

```diff
@@ -12,6 +12,9 @@ PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/fluent-bit.env
 
+ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/flb-storage/
+ExecStartPre=/bin/chown -R dcos_fluent_bit /var/lib/dcos/flb-storage/
```

Comment:
This can also be done as part of the bootstrap, as for Telegraf, which generally allows more control as things get more complicated: https://github.com/dcos/dcos/blob/83d3bb899332a27720ab6087ff0b6062f725b0f6/packages/bootstrap/extra/dcos_internal_utils/cli.py#L139-L163